docs/network-flow-visibility.md

```diff
@@ -56,14 +56,29 @@ ClickHouse as the data storage, and use Grafana as the data visualization and mo
 
 ### Deployment Steps
 
-To deploy the Grafana Flow Collector, the first step is to install the ClickHouse
-Operator, which creates, configures and manages ClickHouse clusters. Check the [homepage](https://github.com/Altinity/clickhouse-operator)
-for more information about the ClickHouse Operator. Current checked-in yaml is based on their
-[v0.18.2](https://github.com/Altinity/clickhouse-operator/blob/refs/tags/0.18.2/deploy/operator/clickhouse-operator-install-bundle.yaml) released version. Running the following command
-will install ClickHouse Operator into `kube-system` Namespace.
+From Theia v0.1, we support deploying the Grafana Flow Collector with Helm. Here
```

Comment:
Since nothing was released before 0.1 for theia we don't need to specify "from theia 0.1"

Reply:
Removed "from theia 0.1".